### DISCUSSION TITLE
Can Fief Admin dashboard be accessible by several users?

### DISCUSSION THREAD

**Discussion by Kh-Oleg** [fief-dev/fief#214] [Q&A] [answered]
It seems that only one user has access to the Admin Dashboard of the given workspace. 
When a service uses Fief for user authorization, it's highly desirable to have more than one 'admin' user for the case, if the first person is not available. 

Is there a possibility (or plans to add such possibility) to let multiple users managing roles & permissions?

**Comment by frankie567** [comment, marked as answer]
Indeed, there is no "user-friendly" way to do this. If you're self-hosted, what you can do is to manually add a row in the `workspace_users` table: one column for the workspace ID, the other one for the user ID. 

I plan to add such feature, but I would have liked to leverage the "Invite user" feature so we could implement it really cleanly.
